Vagrant.md

```diff
@@ -40,9 +40,37 @@ During `vagrant up` and `vagrant halt`, vagrant will attempt to modify your /etc
 Cmnd_Alias VAGRANT_EXPORTS_ADD = /usr/bin/tee -a /etc/exports
 Cmnd_Alias VAGRANT_NFSD = /sbin/nfsd restart
 Cmnd_Alias VAGRANT_EXPORTS_REMOVE = /usr/bin/sed
-Cmnd_Alias VAGRANT_HOSTSUPDATER_ADD = /bin/sh -c echo "*" >> /etc/hosts
-Cmnd_Alias VAGRANT_HOSTSUPDATER_REMOVE = /usr/bin/sed -i -e /*/ d /etc/hosts
-%admin ALL=(root) NOPASSWD: VAGRANT_EXPORTS_ADD, VAGRANT_NFSD, VAGRANT_EXPORTS_REMOVE, VAGRANT_HOSTSUPDATER_ADD, VAGRANT_HOSTSUPDATER_REMOVE
+Cmnd_Alias VAGRANT_HOSTMANAGER_UPDATE = /bin/cp /Users/grahamb/.vagrant.d/tmp/hosts.local /etc/hosts
```

Comment:
If this command is doing what I think it's doing, it won't work on other machines because the user directory will be different.